### PR TITLE
feat: Use laptop Icon from UI icon

### PR DESCRIPTION
**Pull request by Crash--**
Since we have this icon in our Icon's palette, let's use it. 

Also removed the now unused icon

### CODE REVIEW

**Comment by y-lohse** [comment]
But the illustration and the icon aren't the same though? And the new icon doesn't have a color, so it will be black?

**Comment by Crash--** [comment]
Oh right I shouldn't have deleted this device-laptop.svg file since... I used this one.

File: react/PushClientButton/index.jsx [hunk 1]
```diff
@@ -17,7 +16,7 @@ const ButtonClient = props => {
       onClick={onClick}
     >
       <figure>
-        <img width="32" height="32" src={DeviceIcon} />
+        <Icon icon="device-laptop" size="32" />
```
Comment (y-lohse) [inline react/PushClientButton/index.jsx]: Soooo it should be `devices-laptop`, right?

Comment (Crash--) [reply react/PushClientButton/index.jsx]: Nop `device-laptop` is fine ;) https://docs.cozy.io/cozy-ui/react/#!/Icon/1

Comment (drazik) [reply react/PushClientButton/index.jsx]: I think it still works because the sprite was not re-generated, but it shouldn't, no?

Comment (Crash--) [reply react/PushClientButton/index.jsx]: Oh come on! I didn't see my mistake in the filename :o. Thanks :)

**Comment by ptbrowne** [comment]
Ping @Crash-- , is there still work to do ?

**Pull request merged**